orc8r/cloud/go/services/orchestrator/servicers/builder_servicer.go

```diff
@@ -191,3 +198,25 @@ func getVpnMconfig(gatewayConfig *models.MagmadGatewayConfigs) *mconfig_protos.O
 
 	return ret
 }
+
+func getStateMconfig(net configurator.Network, gwKey string) *mconfig_protos.State {
+	ret := &mconfig_protos.State{
```

Comment:
Nit: I don't like using "ret" as a variable, since it obfuscates the contents of the variable / its affordances. Just name it like you would any other variable

Reply:
kk was following the convention in the other helpers in this class but can rename.